website/src/data/users.tsx

```diff
@@ -229,7 +229,16 @@ const Users: User[] = [
   },
 
   //-------  ALL OTHER TEMPLATES go here, can be searched for by name
-
+  {
+    title: 'Python (Django) Web App with PostGreSQL',
```

Comment:
Should be "PostgreSQL"

Reply:
Ooo, I didn't realize that was the correct capitalization. I need to go fix it in a bunch of my own stuff now. :)